docs/docs/how_to/how-to-oracles.md

```diff
@@ -22,7 +22,7 @@ This guide shows you how to use oracles in your Noir program. For the sake of cl
 
 This guide has 3 major steps:
 
-1. How to modify our Noir program to make use of oracle calls as unconstrained functions
+1. How to modify our Noir program to make use of oracle calls as unconstrained function
```

Comment:
```suggestion
1. How to modify our Noir program to make use of oracle calls as unconstrained functions
```

This does not look like a correct edit

Reply:
This looks like where it was meant to be (https://github.com/noir-lang/noir/pull/6598)

Reply:
Thanks!